lib/engine/game/g_2038/corporation.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require_relative '../../corporation'
+
+module Engine
+  module Game
+    module G2038
+      class Corporation < Engine::Corporation
+        attr_accessor :presidents_share
```

Comment:
delete this sub class

Reply:
This wasn't meant to be in yet but it will be needed later because 2038 does capitalisation in a special way.

Reply:
let's chat about this on slack